src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/ErrorPageTests.kt

```diff
@@ -7,12 +7,13 @@ import org.jsoup.nodes.Document
 import org.junit.Test
 import org.vaccineimpact.orderlyweb.db.AppConfig
 import org.vaccineimpact.orderlyweb.db.JooqContext
-import org.vaccineimpact.orderlyweb.db.Tables.REPORT_VERSION
-import org.vaccineimpact.orderlyweb.db.Tables.REPORT_VERSION_ARTEFACT
+import org.vaccineimpact.orderlyweb.db.Tables.*
+import org.vaccineimpact.orderlyweb.db.fromJoinPath
 import org.vaccineimpact.orderlyweb.models.Scope
 import org.vaccineimpact.orderlyweb.models.permissions.ReifiedPermission
 import org.vaccineimpact.orderlyweb.tests.integration_tests.tests.IntegrationTest
 import org.vaccineimpact.orderlyweb.viewmodels.IndexViewModel
+import org.vaccineimpact.orderlyweb.db.joinPath
```

Comment:
it move up with the other `.db` imports? It had looked like a duplicate on first scan

Reply:
No longer needed anyway..